database/migrations/postgres/1738234757185160_remove_custom_migrations.up.sql

```diff
@@ -1 +0,0 @@
-DROP TABLE IF EXISTS custom_migration_cutoff;
```

Comment:
please make sure that all releases (on github) that still have one of these sql files, are set to "prerelease:true"